### ISSUE TITLE
NullPointerException: Attempt to invoke virtual method 'FlurryMessage.getNotificationId()' on a null object reference

### ISSUE THREAD

**Issue by RodolfoGS** [flurry/react-native-flurry-sdk#28]
**Describe the bug**
I have this crash on Crashlytics (Android), I don't know how to reproduce it.

```
Caused by java.lang.NullPointerException
Attempt to invoke virtual method 'int com.flurry.android.marketing.messaging.notification.FlurryMessage.getNotificationId()' on a null object reference
```

**This Bug is About**
Please choose the closest item by replacing `[ ]` with `[x]`.
- [ ] Integration
- [x] Analytics
- [ ] Messaging/Push
- [ ] Remote Configuration
- [ ] Flurry Website/Dashboard
- [ ] Other

**Platform**
Please choose the platform(s) that you are having the issue by replacing `[ ]` with `[x]`.
- [x] Android
- [ ] iOS
- [ ] tvOS

**Environment**
Please tell us the versions of SDKs you are using. If you are not sure about React Native version, you can run `react-native --version` under your project.
* React Native version: 0.63.3
* react-native-flurry-sdk version: 6.1.0

**To Reproduce**
I have this crash on Crashlytics (Android), I don't know how to reproduce it. Sorry.

**Screenshots**
From Crashlytics

**Additional context**
Are you using Flurry Push/Messaging? No
(iOS) Are you using CocoaPods for React Native integration? Yes

**Stacktrace**
```
Fatal Exception: java.lang.RuntimeException: Unable to start receiver com.flurry.android.marketing.messaging.notification.NotificationCancelledReceiver: java.lang.NullPointerException: Attempt to invoke virtual method 'int com.flurry.android.marketing.messaging.notification.FlurryMessage.getNotificationId()' on a null object reference
       at android.app.ActivityThread.handleReceiver(ActivityThread.java:2735)
       at android.app.ActivityThread.-wrap14(ActivityThread.java)
       at android.app.ActivityThread$H.handleMessage(ActivityThread.java:1424)
       at android.os.Handler.dispatchMessage(Handler.java:102)
       at android.os.Looper.loop(Looper.java:148)
       at android.app.ActivityThread.main(ActivityThread.java:5421)
       at java.lang.reflect.Method.invoke(Method.java)
       at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:726)
       at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:616)

Caused by java.lang.NullPointerException: Attempt to invoke virtual method 'int com.flurry.android.marketing.messaging.notification.FlurryMessage.getNotificationId()' on a null object reference
       at com.flurry.android.marketing.messaging.notification.NotificationCancelledReceiver.onReceive(SourceFile:24)
       at android.app.ActivityThread.handleReceiver(ActivityThread.java:2728)
       at android.app.ActivityThread.-wrap14(ActivityThread.java)
       at android.app.ActivityThread$H.handleMessage(ActivityThread.java:1424)
       at android.os.Handler.dispatchMessage(Handler.java:102)
       at android.os.Looper.loop(Looper.java:148)
       at android.app.ActivityThread.main(ActivityThread.java:5421)
       at java.lang.reflect.Method.invoke(Method.java)
       at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:726)
       at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:616)
```

**Device**
* Brand: LGE
* Model: Nexus 4
* Android Version: 6.0.1

**Comment by poting-oath** [comment]
Fix will be available in the next Flurry SDK/Plugin release. I.e., post Flurry Android SDK 12.9.0.

**Comment by poting-oath** [comment]
@RodolfoGS Fix has been released in version 6.3.0. Thanks!
